### DISCUSSION TITLE
Reset after Pinch Zoom

### DISCUSSION THREAD

**Discussion by julian-leahy** [google/model-viewer#2972] [Q&A] [answered]
I was wondering if there is a way to reset the viewer to its initial loaded state after it has been zoomed in using a pinch action. Setting camera orbit back to original values gets me the initial size but does not remove the zoomed in state.

**Comment by elalish** [comment, marked as answer]
You also need to reset the `field-of-view`.

**Reply by julian-leahy** [reply]
Thanks bro!!
